README.md

```diff
@@ -127,6 +127,16 @@ _Black_ is a well-behaved Unix-style command-line tool:
 - it only outputs messages to users on standard error;
 - exits with code 0 unless an internal error occurred (or `--check` was used).
 
+### Using _Black_ with other tools
+
+While _Black_ enforces formatting that conforms to PEP 8, other tools may raise warnings
+about _Black_'s changes or will overwrite _Black_'s changes... I am looking at you
+[isort](https://pypi.org/p/isort). Since _Black_ is barely configurable, these tools
```

Comment:
I don't think we should call out isort directly. It was never designed to work _with_ black. Can we remove this please or reword to be a little more subtle.
"A good example here is isort"

Reply:
Yeah, I agree. In hindsight, isort and _Black_ were never designed to be compatible with each other and it's unfair to call out isort for that. I'll try to reword it to more gentle or just remove if I fail at doing that.